indigo_app/notifications.py

```diff
@@ -115,6 +138,12 @@ def notify_comment_posted(comment_id):
         log.warning("Comment with id {} doesn't exist, ignoring".format(comment_id))
 
 
+@receiver(user_signed_up, sender=User)
+def user_sign_up_signal_handler(**kwargs):
```

Comment:
Similarly to the other PR, please move the signal handler into `signals.py`, with a supporting method here.

Oh, and please include it in the background emails block below.